### PR TITLE
Update Banxa icon and limits

### PR DESCRIPTION
**Pull request by peachbits**

### CODE REVIEW

File: src/modules/Logs/api.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // @flow
 
-const API_PREFIX = 'https://info1.edgesecure.co:8444/v1/'
+const API_PREFIX = 'https://logs1.edge.app:8444/v1/'
```
Comment (paullinator) [inline src/modules/Logs/api.js:3]: not part of this pull request

as well remove the port for logs1

**Review by paullinator — CHANGES_REQUESTED** [review]

File: src/modules/Logs/api.js (hunk 1 above)
Comment (peachbits) [reply src/modules/Logs/api.js]: My branches are polluting each other. Removed.
